Mgf/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```

Comment:
it doesn't look like mzml has an AssemblyInfo.cs, and I know some of this stuff is added by AppVeyor to patch in the correct version. we should look at this a bit more